### PR TITLE
Add crio-status client for convenient HTTP API access

### PR DESCRIPTION
**Pull request by saschagrunert**
The client API is now internal but we can provide a simple command line
interface for it.

### CODE REVIEW

**Comment by vrothberg** [comment]
> Code LGTM but tests and man pages are missing.

And bash-completions would be great.

File: cmd/crio-status/main.go [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+			),
+		},
+	}
+	app.Flags = flags
+	app.Commands = []cli.Command{{
+		Action:  config,
+		Aliases: []string{"c"},
+		Flags:   flags,
+		Name:    "config",
+		Usage:   "retrieve the configuration as TOML string",
```
Comment (TomSweeneyRedHat) [inline cmd/crio-status/main.go]: nit suggestion  s/as TOML/as a TOML/

**Comment by saschagrunert** [comment]
> Code LGTM but tests and man pages are missing.

Added it. :+1: 

> And bash-completions would be great.

Can we tackle this later on? The [urfave/cli](https://github.com/urfave/cli#bash-completion) generator offers only a two-step generation where I'm not sure if we want this... Maintaining seems also not nice to me, WDYT?

File: cmd/crio-status/main.go (hunk 1 above)
Comment (saschagrunert) [reply cmd/crio-status/main.go]: Fixed as suggested.

**Comment by vrothberg** [comment]
> Can we tackle this later on? The [urfave/cli](https://github.com/urfave/cli#bash-completion) generator offers only a two-step generation where I'm not sure if we want this... Maintaining seems also not nice to me, WDYT?

Why not generate it now and add it? I think it's better to have a simple completion than none. If you prefer to add this later, please open an issue so we don't forget about it. My squirrel brain would certainly :^)

**Comment by saschagrunert** [comment]
> Why not generate it now and add it? I think it's better to have a simple completion than none. If you prefer to add this later, please open an issue so we don't forget about it. My squirrel brain would certainly :^)

Added the bash completion and adapted the install scripts. As a little extra I added a fish completion, too! :heart_eyes:

**Pull request merged**